cluster-autoscaler/cloudprovider/azure/azure_client_test.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package azure
+
+import (
+	"net/http"
+	"os"
+	"testing"
+
+	"github.com/Azure/go-autorest/autorest"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMSALAuthorizer(t *testing.T) {
```

Comment:
in the final version of this PR, this should be either removed or replaced with a proper, much more complicated test - atm, this is just a way for you to be able to run it, it's not a test per-se